renku/command/view_model/plan.py

```diff
@@ -69,10 +71,11 @@ def from_input(cls, input: CommandInput, plan_id: Optional[str] = None):
             name=input.name,
             description=input.description,
             default_value=str(input.default_value),
-            position=str(input.position),
+            position=str(input.position) if input.position else None,
```

Comment:
`if input.position is not None`!? `position` can be 0 too, although it might not be assigned to parameters. I still think we should check against `None` since the value is an integer.